app/services/travel_to_work_area_and_london_borough_service.rb

```diff
@@ -1,20 +1,60 @@
 class TravelToWorkAreaAndLondonBoroughService
-  def self.add_travel_to_work_area_and_london_borough(site:)
-    url = URI("#{Settings.mapit_url}/point/4326/#{site.longitude},#{site.latitude}?type=TTW,LBO&api_key=#{Settings.mapit_api_key}")
-    response = Net::HTTP.get(url)
-    json = JSON.parse(response)
-
-    ttw_area = json.dig(json.keys.first, "name")
-
-    london_borough = if ttw_area == "London"
-                       json
-                       .dig(json.keys.second, "name")
-                       .gsub(/ Borough Council| City Council| Corporation/, "")
-                     end
-
-    site.update_column("travel_to_work_area", ttw_area)
-    site.update_column("london_borough", london_borough)
-  rescue StandardError
-    false
+  include ServicePattern
+
+  def initialize(site:)
+    @site = site
+  end
+
+  def call
+    ttw_area = get(:travel_to_work_area)
+
+    if ttw_area == "London"
+      london_borough = get(:london_borough)
+
+      if london_borough
+        site.update_column("travel_to_work_area", "London")
+        site.update_column("london_borough", london_borough)
+        site.save!(validate: false)
+      end
+    elsif ttw_area
+      site.update_column("travel_to_work_area", ttw_area)
+      site.update_column("london_borough", nil)
+      site.save!(validate: false)
```

Comment:
`site.save!(validate: false)`
What kind of validation are you bypassing?
Is an actual need to bypass it?
What is the user case of `update_column`/`update_columns` given that it itself already bypass validation?

https://api.rubyonrails.org/v6.1.0/classes/ActiveRecord/Persistence.html#method-i-update_columns

If the site was never valid... then this should be a no ops?

https://github.com/DFE-Digital/teacher-training-api/blob/4463c0ec77df7a0659340e55b88837f4d6533a9b/app/models/site.rb#L18-L26

Reply:
@defong If I recall, `validate: false` is there because of imported legacy sites where address data may be dodgy? Same approach is applied with geocoding

Reply:
I think we originally applied the validations but later the decision was taken (Paul Hayes possibly?) 
to geocode all sites regardless because too many were being excluded

Reply:
But yes I think we should be doing

```
  site.travel_to_work_area = "London"
  site.london_borough = london_borough
  site.save!(validate: false)
 ```